src/ProgressLogging.jl

```diff
@@ -451,46 +451,32 @@ function _progress(name, thresh, ex)
 end
 
 function _progress(name, thresh, ex, target, result, loop, iter_vars, ranges, body)
-    count_vars = [Symbol("i$k") for k = 1:length(iter_vars)]
-    iter_exprs = [:(($i, $(esc(v))) = enumerate($(esc(r)))) for (i, v, r) in zip(
+    count_vars = [gensym(Symbol("i$k")) for k = 1:length(iter_vars)]
+    iter_exprs = [:(($i, $v) = $enumerate($r)) for (i, v, r) in zip(
         count_vars,
         iter_vars,
         ranges,
     )]
-    _id = "progress_$(gensym())"
+    @gensym count_to_frac val frac lastfrac
```

Comment:
See https://github.com/JunoLab/ProgressLogging.jl/pull/13#discussion_r372223440 for why this _horrible_ manual `@gensym` and aggressive interpolation below are required.  I don't know if there is a better way to do this...